### PR TITLE
Update documentation for macOs

### PR DESCRIPTION
**Pull request by aarhusgregersen**
- Recommend installing VS Code extensions (as debug script won't work without it)
- Move around documentation in Step 4, as currently it isn't an optional note but a requirement
- Add clarity to bullet 1, in Step 4.
- Added additional information in Step 3 info box

These are minor but I lacked that information when starting, so figured someone else might too.

### CODE REVIEW

File: essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-macos.md [hunk 1]
```diff
@@ -30,16 +31,22 @@ Skip this if flutter is already installed on your system.
...
+> ```
+> Make sure to restart your terminal and VS Code
 
 ## **Step 4: Edit and run the application**
 
-1. Open the `frontend` folder located at xx/AppFlowy/frontend with VS Code.
-2. Go to the Run and Debug tab and then click AF: Clean + Rebuild All for the first time running.
+1. Open the `frontend` folder located at xx/AppFlowy/frontend with VS Code. It is important *not* to open the root folder, as that will not give access to the appropriate debug commands.
+2. Check the device selection, as of now AppFlowy only supports Desktop: 
+![device](https://user-images.githubusercontent.com/86001920/144546864-cebbf0c0-4eef-424e-93c7-e1e6b3a59669.png)
```
Comment (vincentdchan) [inline essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-macos.md:45]: Please upload your image to `../../../../.gitbook/assets/`

**Comment by appflowy** [comment]
@aarhusgregersen Exporting the path is a significant step. Thank you for contributing!

File: essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-macos.md (hunk 1 above)
Comment (aarhusgregersen) [reply essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-macos.md:45]: I believe it is already there @vincentdchan I simply moved around an image that was already in the documentation. Am I wrong? 😄

Comment (vincentdchan) [reply essential-documentation/contribute-to-appflowy/software-contributions/environment-setup/building-on-macos.md:45]: True. It seems that it's a legacy problem.

**Pull request merged**